conda_devenv/devenv.py

```diff
@@ -111,7 +115,7 @@ def handle_includes(root_filename, root_yaml):
                         included_filename=included_filename, filename=filename
                     )
                 )
-            queue[included_filename] = included_yaml_dict
+            queue[str(included_filename.resolve())] = included_yaml_dict
```

Comment:
No need to call `resolve()` here: we already changed the path to absolute form here:

```python
            if Path(included_filename).is_absolute():
                included_filename = Path(included_filename)
            else:
                included_filename = Path(filename).parent / included_filename
```

Reply:
`str(included_filename.resolve())` must stay because `included_filename = Path(included_filename)` can have thing like `/a/../b/c` the will resolve to `/b/c`.

Reply:
Actually needed, to resolve relative paths when adding to queue